### DISCUSSION TITLE
File Upload

### DISCUSSION THREAD

**Discussion by patilsuguresh** [community/community#64393] [Repositories]
### Select Topic Area

General

### Body

Cannot upload file in repository, getting error.

**Comment by mutasim77** [comment]
First create repo then follow these commands: 
-  1: open (terminal/cmd) and type git .init (this initializes empty git folder)
- 2: git remote add origin https://..... (copy the http url of your repo)
- 3: git add . (this adds all the changed file for commit)
- 4: git commit -m "initial commit" (commit command ,commits the code locally)
- 5: git push origin master(this will push the code to server(github))

**Comment by iamsnehadas** [comment]
If you are not using command line, then, you can also try the following steps to upload a file:

- Create a repository
- Click on the drop-down arrow of add file
- Click on upload files
- You can drag your files to the specified box or you can choose your file to upload them.
- After the file is uploaded, write a message and click on commit changes.

If you are facing some other issues, then please specify it.
